schema/JF_TRAN.json

```diff
@@ -295,19 +296,18 @@
     "contribution_date": {
       "title": "CONTRIBUTION DATE",
       "description": "",
-      "type": "integer",
-      "minimum": 0,
-      "maximum": 99999999,
-      "examples": [
-        20120615
-      ],
+      "type": "string",
+      "minLength": 8,
+      "maxLength": 8,
+      "pattern": "^\\d{8}$",
+      "examples": [20120615],
       "fec_spec": {
         "COL_SEQ": 21,
         "FIELD_DESCRIPTION": "CONTRIBUTION DATE",
         "TYPE": "NUM-8",
         "REQUIRED": "X (error)",
         "SAMPLE_DATA": 20120615,
-        "VALUE_REFERENCE": "YYYYMMDD",
+        "VALUE_REFERENCE": null,
```

Comment:
@mjtravers is it our intention to have the fec_spec property match the original fec spreadsheet or our update one?

Reply:
Awesome! I'll keep that in mind next time I edit a schema